object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -73,6 +78,9 @@ public void postAnalyze(ControlFlowGraph cfg) {
   List<String> getMustCallValue(Tree tree) {
     MustCallAnnotatedTypeFactory mustCallAnnotatedTypeFactory =
         getTypeFactoryOfSubchecker(MustCallChecker.class);
+    if (mustCallAnnotatedTypeFactory == null) {
```

Comment:
How is this change related to temporary variables?  Same for the next change below

Reply:
I still have this question

Reply:
I added this check to handle `NullPointerException` [here](https://github.com/kelloggm/object-construction-checker/blob/temporaryVar/object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionTransfer.java#L103) when the given `node` is `(this).<init>()`. I got a lot of `NullPointerException` in AutoValue test cases.